### ISSUE TITLE
podman/docker container

### ISSUE THREAD

**Issue by micressor** [lovvtide/satellite-web#11]
i would be very happy about a podman/docker container that I can easily run.

**Comment by xeruf** [comment]
yes, I concur, would love to have an all-in-one docker-compose setup including relay, client & proxy :)
